### ISSUE TITLE
Colorblind mode

### ISSUE THREAD

**Issue by nodiscc** [RequestPolicyContinued/requestpolicy#689]
I was told that green and red items in the menu look almost the same color to people with a common variant of color blindness. This could affect 5-8% of users: https://en.wikipedia.org/wiki/Color_blindness#Epidemiology. Can someone confirm?

There should be a colorblind mode switch that switches all green items to a blue tint. This should be proposed during initial setup https://github.com/RequestPolicyContinued/requestpolicy/issues/545

**Comment by myrdd** [comment]
> Can someone confirm?

What do you want to have confirmed?

I first need to learn about colorblindness. I'm happy if someone wants to explain to me what needs to be changed exactly. I've found [this article](http://designshack.net/articles/accessibility/tips-for-designing-for-colorblind-users/) on how to design for colorblind users.
